README.md

```diff
@@ -1,113 +1 @@
-   * source files (excluding build files)
- * Build status
-   * Success
-   * Failure
-
-The cache will be saved in two states: 
-
- * a _success_ cache is saved on the first successful build after the build files has been updated.
- * a _failure_ cache is saved on the first failed build after the build files has been updated, if a corresponding _success_ cache does not already exists.
-
-So in other words, when the build files are updated, a cache is always created. Ununsed dependencies are purged before saving the _success_ cache.
-
-If the `.circleci/config.yml` or Gradle wrapper version is updated, the cache is wiped.
-
-### Single-commit use-cases
-
-##### Single commit, single-step workflow
-
-| Commit  | Build status | Expected outcome |
-| ------------- | ------------- | -- |
-| Source files  | Success  | Previous _success_ or _failure_ cache restored, no new cache created |
-| Source files  | Failure  | Previous _success_ or _failure_ cache restored, no new cache created |
-| Build files  | Success  | Previous _success_ or _failure_ cache restored, ununsed dependencies purged, new _success_ cache created |
-| Build files, source files  | Success  | Previous _success_ or _failure_ cache restored, ununsed dependencies purged, new _success_ cache created |
-
-##### Single commit, multi-step workflow
-
-| # | Commit | Step | Depends on | Build status | Expected outcome |
-| ------------- | -- | -- |------------- | -- | -- |
-| 1. | Build files  | STEP1 | - | Success  | Previous _success_ or _failure_ cache restored, new ___success_ cache D__ created |
-|  |   | STEP2  |  STEP1 | Success | ___success_ cache #D__ restored, no new cache created |
-
-### Multi-commit use-cases
-##### Bumping dependencies breaks compilation, fixes.
-
-| # | Commit | Build status | Expected outcome |
-| ------------- | ------------- | -- | -- |
-| 1. | Build files  | Failure  | Previous _success_ or _failure_ cache restored, new ___failure_ cache A__ created |
-| 2. | Source files  | Failure  | ___failure_ cache #A__ restored, no new cache created |
-| 3. | Source files  | Success  | ___failure_ cache #A__ restored, ununsed dependencies purged, new ___success_ cache #B__  created |
-| 4. | Source files  | Success  | ___success_ cache #B__ restored, no new cache created |
-
-##### Bumping dependencies, break unit tests later
-
-| # | Commit | Build status | Expected outcome |
-| ------------- | ------------- | -- | -- |
-| 1. | Build files  | Success  | Previous _success_ or _failure_ cache restored, new ___success_ cache C__ created |
-| 2. | Source files  | Failure  | ___success_ cache #C__ restored, no new cache created |
-
-
-## Troubleshooting
-If the cache is corrupted, update the cache key, so that the previous state is not restored - as in the official Gradle orb.
-
-[issue-tracker]:               https://github.com/entur/gradle-orb
-[CircleCI Gradle]:             https://circleci.com/orbs/registry/orb/circleci/gradle?version=2.2.0
-
-
+Test project
```

Comment:
Wrong file? Was this change intended for `sample_app/README.md`?

Reply:
Reverting